From 3656fd013e40c16d4b4831e3a37273a03c3df01d Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Tue, 6 Oct 2026 22:05:15 +0100
Subject: [PATCH 1/1] Remove duplicate desktop-id launchable in appstream
 metainfo

Since appstream >= 1.2.x having multiple desktop-ids will cause
appstream composition to fail.
---
 org.kde.kclock.appdata.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/org.kde.kclock.appdata.xml b/org.kde.kclock.appdata.xml
index 9dcb9aee..f5003d7e 100644
--- a/org.kde.kclock.appdata.xml
+++ b/org.kde.kclock.appdata.xml
@@ -188,7 +188,6 @@
   <requires>
     <display_length compare="ge">360</display_length>
   </requires>
-  <launchable type="desktop-id">org.kde.kclock.desktop</launchable>
   <screenshots>
     <screenshot type="default">
       <caption>Mobile view</caption>
-- 
2.56.0

